input/docs/handbook/collections/index.md

```diff
@@ -82,14 +82,124 @@ var myBindingOperation = mySource
 
 The API for the above is the same for cache and list.
 
-## So what's the difference between an observable list and an observable cache
+## So what's the difference between a SourceList and a SourceCache
 
-I get asked this question a lot and the answer is really simple.  If you have a unique id, you should use
-an observable cache as it is dictionary based which will ensure no duplicates can be added and it notifies on adds, updates and removes, whereas list allows duplicates and only has no concept of an update.
+If you have a unique id, you should use an observable cache as it is dictionary based which will ensure no duplicates can be added and it notifies on adds, updates and removes, whereas list allows duplicates and only has no concept of an update.
```

Comment:
Probably worth mentioning there are performance advantages to cache.

Reply:
Also worth noting that if you can use `SourceCache` you should use `SourceCache`